source/tyr/tyr/default_settings.py

```diff
@@ -2,13 +2,14 @@
 import logging
 from datetime import timedelta
 from celery import schedules
+import os
 
 # URL for the brokker, by default it's the local rabbitmq
 # For amqp (rabbitMQ) the syntax is:
 # amqp://<user>:<password>@<host>:<port>/<vhost>
 # the default vhost is "/" so the URL end with *two* slash
 # http://docs.celeryproject.org/en/latest/configuration.html#std:setting-BROKER_URL
-CELERY_BROKER_URL = 'amqp://guest:guest@localhost:5672//'
+CELERY_BROKER_URL = os.getenv('CELERY_BROKER_URL', 'amqp://guest:guest@localhost:5672//')
```

Comment:
```diff
- CELERY_BROKER_URL
+ TYR_CELERY_BROKER_URL
```
To be more consistent with the other options and to avoid clashing with other env var.

Reply:
:heavy_check_mark: